src/jit_runner.cpp

```
@@ -0,0 +1,373 @@
+    // Lookup and run __ry_main__
+    auto symOrErr = jit->lookup("__ry_main__");
+    if (!symOrErr) {
+        errs() << "Failed to lookup __ry_main__: ";
+        logAllUnhandledErrors(symOrErr.takeError(), errs());
+        ry::emitTraceEvent("runtime.error", "jit", &sourceLoc,
+                           {ry::TraceField("detail", "Failed to lookup __ry_main__")});
+        return 1;
+    }
+    auto *fn = symOrErr->toPtr<int(*)()>();
+
+    if (test_mode) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)